### PR TITLE
Amit feature/scanner result

### PR DESCRIPTION
**Pull request by amit-metri**
Scanner result screen UI is designed and logic is placed to show vaccinated, partially vaccinated and no records

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -60,6 +64,7 @@ dependencies {
 
     //Dagger Hilt
     implementation "com.google.dagger:hilt-android:$versions.hilt"
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```
Comment (pinakinkansara) [inline app/build.gradle:67]: Can you please check, on this dependency. I believe we don't require this and it is safe to remove it.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -17,7 +17,8 @@
         <activity
             android:name=".MainActivity"
             android:exported="true"
-            android:screenOrientation="portrait">
+            android:screenOrientation="portrait"
+            android:theme="@style/Theme.MySplash">
```
Comment (pinakinkansara) [inline app/src/main/AndroidManifest.xml:21]: Change the name of theme to "Theme.BC_VAX.Splash"

File: app/src/main/java/ca/bc/gov/health/ircreader/ui/scanresult/BarcodeScanResultFragment.kt [hunk 3]
```diff
@@ -1,20 +1,100 @@
...
 class BarcodeScanResultFragment : Fragment(R.layout.fragment_barcode_scan_result) {
 
+    /*
+    * Both of these variable control the UI for status screen
+    * vaccinationStatus = 0 is "No records found"
+    * vaccinationStatus = 1 is "Partially vaccinated"
+    * vaccinationStatus = 2 is "Fully vaccinated"
+    * */
+    private var userName : String = ""
+    private var vaccinationStatus : Int = 0
```
Comment (pinakinkansara) [inline app/src/main/java/ca/bc/gov/health/ircreader/ui/scanresult/BarcodeScanResultFragment.kt:32]: All this logic should go to view model and we have to observe the data to reflect the changes in the fragment.

**Comment by amit-metri** [comment]
Updated code .

**Pull request closed without merging**